### ISSUE TITLE
Let you know if you don't, there is a `faster whisper` than `whisper.cpp`

### ISSUE THREAD

**Issue by HaujetZhao** [Const-me/Whisper#143]
Another repo [faster-whisper](https://github.com/guillaumekln/faster-whisper) that uses [CTranslate2](https://github.com/OpenNMT/CTranslate2/) to inference is n times faster than [whisper.cpp](https://github.com/ggerganov/whisper.cpp). And it supports word-level timestamp. 

Just let you know this in case you don't. 

And, is it possible to make [faster-whisper](https://github.com/guillaumekln/faster-whisper) use DirectX to accelerate more?

**Comment by emcodem** [comment]
Yeah i have seen this project, it would be interesting if anyone could invest some serious effort to compare faster-whisper and const-me version in terms of speed and quality. 
Also i wonder if these kind of projects that try to be backward compatible to original whisper commandline usage already have a mode for "live" transcription as well?

**Comment by HaujetZhao** [comment]
https://github.com/guillaumekln/faster-whisper/issues/151  Mentioned the realtime transcription, seems work using vad.

**Comment by ghost** [comment]
I'm a tech noob so take my comment with a grain of salt. I have used both and currently Const-me's version produces better results.

**Comment by emcodem** [comment]
@softlypink 
faster-whisper with Ctranslate2 has lots of magic builtin regarding the sampling strategy, but Const only has simple greedy search (which i think is totally fine for use with whisper model). 
For better comparison, try to disable all related logic e.g. using beam_size=1,best_of=1, temperature=0. Penalty and log_prop also seem to play a role but i didnt quickly find out how to disable them.


This way output should be more compareable but their strategy to salvage repeated output (using the temperature parameter) is also disabled so you'll get more loops again. It's a pity.

**Comment by Murugurugan** [comment]
This should get the highest priority, after all having faster-whisper added speed + support on AMD would be a huge win for everyone.
